Explorer/Assets/DCL/SDKComponents/RealmInfo/WriteRealmInfoSystem.cs

```diff
@@ -0,0 +1,57 @@
+using Arch.Core;
+using Arch.SystemGroups;
+using CrdtEcsBridge.Components;
+using CrdtEcsBridge.ECSToCRDTWriter;
+using DCL.ECSComponents;
+using DCL.Multiplayer.Connections.RoomHubs;
+using DCL.Utilities;
+using ECS;
+using ECS.Abstract;
+using ECS.Groups;
+
+namespace DCL.SDKComponents.RealmInfo
+{
+    [UpdateInGroup(typeof(SyncedInitializationSystemGroup))]
+    public partial class WriteRealmInfoSystem : BaseUnityLoopSystem
+    {
+        private readonly IECSToCRDTWriter ecsToCRDTWriter;
+        private readonly ObjectProxy<IRealmData> realmDataProxy;
+        private readonly ObjectProxy<IRoomHub> roomHubProxy;
+
+        internal WriteRealmInfoSystem(World world, IECSToCRDTWriter ecsToCRDTWriter, ObjectProxy<IRealmData> realmDataProxy, ObjectProxy<IRoomHub> roomHubProxy) : base(world)
+        {
+            this.ecsToCRDTWriter = ecsToCRDTWriter;
+            this.realmDataProxy = realmDataProxy;
+            this.roomHubProxy = roomHubProxy;
+        }
+
+        protected override void Update(float t)
```

Comment:
You should propagate data `OnInitialize` as well: some scene/SDK logic may expect this on the frame 0 as we already saw in the `WriteEngineInfo`

Reply:
tackled here: https://github.com/decentraland/unity-explorer/pull/838/commits/62fc44d7e74b697600b976ca9780526b53457f90